source/LumiCalReco/src/LumiCalClusterer_getCalHits.cpp

```diff
@@ -78,7 +78,9 @@ int LumiCalClustererClass::getCalHits(	EVENT::LCEvent * evt,
 
       } else {
 	arm =(*_mydecoder)( calHitIn )["barrel"]; // from 1 and 2
-	phiCell = (*_mydecoder)( calHitIn )["phi"];
+	//FIXME: add check for phi range?
+	phiCell = (*_mydecoder)( calHitIn )["phi"]; // goes from -phiMax/2 to +phiMax/2
+	if(phiCell < 0 ) phiCell += _cellPhiMax; // need to put into positive range only
```

Comment:
I wonder how cellId can depend on rotations of LumiCal, cellId must be calculated using local
hit position. So it means that the transformation of the globalHitPosition  to localHitPosition is simply screwed up in the SD driver, question is wheter this is only LumiCal problem ?